util/constants.js

```diff
@@ -3,6 +3,7 @@ export const AUTH0_SCOPE = ''
 export const ADMIN_USER_URL = `${process.env.API_BASE_URL}/api/admin/user`
 export const API_USER_URL = `${process.env.API_BASE_URL}/api/secure/application`
 export const OTP_USER_URL = `${process.env.API_BASE_URL}/api/secure/user`
+export const OTP_PLAN_URL = `${process.env.API_BASE_URL}/otp/routers/default/plan`
```

Comment:
The other way around to access process vars (for API documentation and email) is to cache them here, and they will be be filled at compile time. That's probably the easiest workaround.

Reply:
I think the issue is what I mentioned here: https://github.com/ibi-group/otp-admin-ui/pull/11#discussion_r480391565